badge/badge.css

```diff
@@ -0,0 +1,52 @@
+@import url(../style.css);
+@import url(../avatar/avatar.css);
+
+.icon-with-badge {
+    position: relative;
+    width: fit-content;
+}
+
+.badge {
+    position: absolute;
+    top: -0.7rem;
+    right: -0.7rem;
+    color: white;
+    height: 1.25rem;
+    width: 1.5rem;
+    padding: 0.375rem;
+    text-align: center;
+    border-radius: 50%;
+}
+
+.red {
+    background-color: var(--danger);
+}
+
+.blue {
+    background-color: var(--secondary);
```

Comment:
good to use color name for color instead of property.

Reply:
yes, I need to change the colors name in root CSS file